app/code/Magento/Sales/Controller/AbstractController/Reorder.php

```diff
@@ -57,6 +57,14 @@ public function execute()
         /** @var \Magento\Framework\Controller\Result\Redirect $resultRedirect */
         $resultRedirect = $this->resultRedirectFactory->create();
 
+        $reorderHelper =  $this->_objectManager->get(\Magento\Sales\Helper\Reorder::class);
+        if (!$reorderHelper->canReorder($order->getEntityId())) {
+            $this->messageManager->addErrorMessage(__("Reorder is disabled."));
```

Comment:
May you change the phrase to the 'Reorder not available.' ?

Reply:
@swnsma - i have changed the phrase